chalice/deploy/deployer.py

```diff
@@ -169,6 +177,104 @@ def _validate_manage_iam_role(config):
             )
 
 
+class ChaliceDeploymentError(Exception):
+    def __init__(self, error):
+        # type: (Any) -> None
+        where = self._get_error_location(error)
+        msg = self._wrap_text(
+            'ERROR - %s, received the following error:' % where
+        )
+        msg += '\n\n'
+        msg += self._wrap_text(self._get_error_message(error), indent=' ')
+        msg += '\n\n'
+        suggestion = self._get_error_suggestion(error)
+        if suggestion:
```

Comment:
It should not matter if the check changes here. The output will stay the same, but sure I can update.